src/main/java/com/simpligility/maven/plugins/android/phase04processclasses/ProguardMojo.java

```diff
@@ -113,8 +112,8 @@
     private String[] parsedOptions;
 
     /**
-     * Path to the proguard jar and therefore version of proguard to be used. By default this will load the jar from
-     * the Android SDK install. Overriding it with an absolute path allows you to use a newer or custom proguard
+     * Path to the proguard jar and therefore version of proguard to be used. By default this will use Proguard 5.2.1
```

Comment:
Shouldn't specify the version in the comment, or the comment will have to be updated every time the dependency is. "use Proguard from the Central Repository" should be fine.